tokio/Cargo.toml

```diff
@@ -85,6 +84,11 @@ sync = []
 test-util = ["rt", "sync", "time"]
 time = []
 
+# Technically, removing this is a breaking change even though it only ever did
+# anything with the unstable flag on. It is probably safe to get rid of it after
+# a few releases.
+stats = []
```

Comment:
* If we rename to `metrics` we will need to change the feature flag. While unstable, I'm not worried about it since technically adding a feature flag is part of the public API.
* We can always do runtime enabling / disabling of metrics collection.